app/public/MovementBar.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <style>
+            #progress-bar {
+                width: 100%;
+                height: 20px;
+                background-color: lightseagreen;
+                text-align: center;
+                line-height: 25px;
+                border-radius: 30px;
+                color: black;
+                font-weight: bolder;
+            }
+        </style>
+    </head>
+    <body>
+        <h1>Testing Progress Bar</h1>
+
+        <div id="current-progress">
+            <div id="progress-bar"></div>
+        </div>
+
+        <br>
+        <button onclick="PROGRESS_BAR(document.getElementById('progress-bar'))">Run Progress</button>
```

Comment:
This is making a new PROGRESS_BAR object every time you click the button, which is probably not what you intended. You should make it only once, then call some function on it each time you want it to start

Reply:
In the html:
```
<button onclick="bar.startFill()">Run Progress</button>
```

in the script:
```
const bar = PROGRESS_BAR(document.getElementById('progress-bar'))
```